config/locales/en.yml

```
@@ -121,6 +121,15 @@ en:
     mathematics:
       body: Institute of Mathematics and its Applications
       url: https://teachingmathsscholars.org/eligibilitycriteria
+    french:
+      body: British Council
```

```suggestion
body: &british_council British Council
```

then you can reuse as:

```
german:
      body: *british_council
```

---

Fixed